hubverse-aws-upload/hubverse-aws-upload.yaml

```diff
@@ -57,15 +57,24 @@ jobs:
           'hub-config'
           'model-abstracts'
           'model-metadata'
+          # temporarily include model-output here
           'model-output'
           'target-data'
         )
         for DIRECTORY in "${hub_directories[@]}"
         do
-          if [ -d "./$DIRECTORY" ]; then aws s3 sync "./$DIRECTORY" "s3://$BUCKET_NAME/$DIRECTORY" --delete; fi
+          if [ -d "./$DIRECTORY" ]
+          then
+            rclone sync \
+              "./$DIRECTORY/" \
+              ":s3,provider=AWS,env_auth:$BUCKET_NAME/$DIRECTORY" \
+              --checksum --verbose --stats-one-line --config=/dev/null
```

Comment:
`--checksum` instructs rclone to use file size + hash/checksum when determining whether or not a file has changed (instead of file size + update date)

`--config=/dev/null` is a hack to suppress output about not finding a config file (we don't use one here, since we're not using any special setup options)

more info:
https://rclone.org/commands/rclone_sync/#copy-options